src/main/java/de/propra/timetracker/TimeTrackerCLI.java

```diff
@@ -37,8 +37,12 @@ CLIStatus readCLI(String[] args) throws IOException {
         options.addOption(addOption);
         Option sumOption = new Option(null, "sumof", true, "Summiere eingegebene Einträge eines bestimmten Projektes");
         options.addOption(sumOption);
+        Option sumOptionDate = new Option(null, "sumofd", true, "Summiere eingegebene Einträge eines bestimmten Datums");
```

Comment:
Wie wäre es mit `sum-of-day` statt der unleserlichen Abkürzung?

Reply:
Kay und vieleich sum-of-project für die andere sumof funktion

Reply:
Ja genau :-)